Sources/tart/Root.swift

```diff
@@ -39,6 +39,14 @@ struct Root: AsyncParsableCommand {
       SentrySDK.start { options in
         options.dsn = dsn
         options.releaseName = CI.release
+        options.tracesSampleRate = 1.0
+
+        // By default only 5XX are captured
+        // Let's capture everything but 401 (unauthorized)
+        options.failedRequestStatusCodes = [
```

Comment:
Should we also set `enableCaptureFailedRequests` for this to work?

Reply:
I think it's on by default but can certainly explicitly add it. 👌